### ISSUE TITLE
[models] Create specific model's heads for binary classification

### ISSUE THREAD

**Issue by frgfm** [pyronear/pyro-vision#19]
## 🚀 Feature
Create a `models.utils` submodule with a function to easily create model heads depending on the type of task, namely binary classification here.

## Motivation
Current training scripts in `references` assume that the target is categorical (multi-label) rather than binary. A proper task setting would allow the model to better optimize during training.

## Pitch
The function would take as arguments:
- model
- task type
- options for head design (dropout, bn, concatenated pooling, etc.)

and return a `torch.nn.Sequential` object.

**Comment by frgfm** [comment]
Closed by #30 & #37
